src/docs/manual/index.html

```diff
@@ -632,119 +632,90 @@ <h3 id="command-line-options">Command-line options</h3>
- This class is tested in
- addition to any specified using <code>--classlist</code>, and must be
- accessible from the package of the tests (set with
- <code>--junit-package-name</code>).
+             The fully-qualified name of a class to test; for example, <code>--testclass=java.util.TreeSet
+ </code>. All of its methods are methods under test. This class is tested in addition to any
+ specified using <code>--classlist</code>, and must be accessible from the package of the tests
+ (set with <code>--junit-package-name</code>).
             <li id="option:classlist"><b>--classlist=</b><i>filename</i>.
-             File that lists classes to test.
- All of their methods are methods under test.
-
- In the file, each class under test is specified by its fully-qualified name
- on a separate line. See an <a href=
- "https://randoop.github.io/randoop/manual/class_list_example.txt">
- example</a>. These classes are tested in addition to any specified using
- <code>--testclass</code>. All classes must be accessible from the package of
- the tests (set with <code>--junit-package-name</code>).
+             File that lists classes to test. All of their methods are methods under test.
+
+ <p>In the file, each class under test is specified by its fully-qualified name on a separate
+ line. See an <a href= "https://randoop.github.io/randoop/manual/class_list_example.txt">
+ example</a>. These classes are tested in addition to any specified using <code>--testclass
+ </code>. All classes must be accessible from the package of the tests (set with <code>
+ --junit-package-name</code>).
             <li id="option:methodlist"><b>--methodlist=</b><i>filename</i>.
              File that lists methods to test.
 
- In the file, each each method under test is specified on a separate line.
- The list of methods given by this argument augment any methods determined via
- the <code>--testclass</code> or <code>--classlist</code> option.
+ <p>In the file, each each method under test is specified on a separate line. The list of
+ methods given by this argument augment any methods determined via the <code>--testclass</code>
+ or <code>--classlist</code> option.
 
- <p>
- A constructor line begins with <code>"cons :"</code> followed by the
- classname, the string <code>&lt;init&gt;</code>, and the constructor's
- parameter types enclosed in parentheses. Methods are specified in a
- similar way. For example:
- </p>
+ <p>A constructor line begins with <code>"cons :"</code> followed by the classname, the string
+ <code>&lt;init&gt;</code>, and the constructor's parameter types enclosed in parentheses.
+ Methods are specified in a similar way. For example:
 
  <pre class="code">
  cons : Type0.&lt;init&gt;(Type1, Type2, ..., TypeN)
  method : Type0.method_name(Type1, Type2, ..., TypeN)
  </pre>
 
- <p>
- Each <code>Type<i>i</i></code> must be fully-qualified (include package
- names).
- </p>
+ <p>Each <code>Type<i>i</i></code> must be fully-qualified (include package names).
```

Comment:
There are many gratuitous changes to the manual (the index.html file), such as this one.  Please undo all of them.  They make it hard to find the real changes in the diffs, and they will pollute the version control history if merged into master.